docs/extend/extend-desktop/mvd-server-config.md

```diff
@@ -86,56 +55,51 @@ ZWED_logLevels_org____zowe____terminal____tn3270_x2e_x2a:"5"
 
-**NOTE: All examples within are based off of the *zlux-app-server* repository.**
 
-## Network configuration
-**Attributes listed here are to be defined within the server's JSON configuration file**
-The App Server can be accessed over HTTP and/or HTTPS, provided it has been configured for either. 
+Although overridden by both environment variables and CLI arguments, for convenience the App server and ZSS read from a configuration file file with a common structure. ZSS reads this directly as a startup argument, while the App Server as defined in the [zlux-server-framework](https://github.com/zowe/zlux-server-framework) repository accepts several parameters which are intended to be read from a JSON file through an implementer of the server, such the default provided in the [zlux-app-server](https://github.com/zowe/zlux-app-server) repository, namely the [lib/zluxServer.js](https://github.com/zowe/zlux-app-server/blob/v2.x/master/lib/zluxServer.js) file. This file accepts a JSON file that specifies most if not all parameters needed, but some other parameters can be provided via flags if desired. 
 
-### HTTP
-To configure the server for HTTP, define an attribute *http* within the top-level *node* attribute. Then, define the following within *http*:
-- *ipAddresses*: An array of strings of ipv4, ipv6, or hostnames that the server will bind to. If not specified, the default is 0.0.0.0
-- *port*: An integer parameter for the TCP port that the server will listen on. Can be 80 or a value between 1024-65535.
-
-### HTTPS
-For HTTPS, a few parameters are required. First, define an attribute *https* within the top-level *node* attribute. Then, define the following within *https*:
-- *ipAddresses*: An array of strings of ipv4, ipv6, or hostnames that the server will bind to. If not specified, the default is 0.0.0.0
-- *port*: An integer parameter for the TCP port that the server will listen on. Can be 443 or a value between 1024-65535.
-- *certificates*: An array of strings which are paths to PEM format HTTPS certificate files.
-- *keys*: An array of strings which are paths to PEM format HTTPS key files.
-- *pfx*: A string which is a path to a PFX file which must contain certificates, keys, and optionally Certificate Authorities.
-- *certificateAuthorities* (Optional): An array of strings which are paths to certificate authorities files.
-- *certificateRevocationLists* (Optional): An array of strings which are paths to certificate revocation list (CRL) files.
-
-**NOTE: When using HTTPS, you must specify *pfx*, or both *certificates* and *keys*.**
```

Comment:
The above are important properties that can be specified in configuration file, did we document them somewhere? Otherwise I wouldn't want to lose that

Reply:
Yes, the json schema.